podcasts/Onboarding/Import/ImportDetailsView.swift

```diff
@@ -1,7 +1,17 @@
 import SwiftUI
 
 struct ImportDetailsView: View {
+    enum OPMLImportResult {
+        case none
+        case success
+        case failure
+    }
```

Comment:
[nit] Wdyt about add breaklines to keep the `enum` separated from the `@Environment` and then another breakline separating the `@State`?

Reply:
updated